### PR TITLE
More improvements

### LINKED ISSUE #4186: V3: `toast.success.password_updated` and `...user_deleted(?)` are missing
**Describe the bug**
There seems to be a string missing for `toast.success.password_updated`.

**To Reproduce**
1. Change password

**Actual behavior**
image

### LINKED ISSUE #4195: V3: Broken presentation deletion button
image

### LINKED ISSUE #4199: V3: Show roasters a bit longer
Roasters show up only very briefly. Would be nice if they would show up a bit longer.

### LINKED ISSUE #4200: V3: Redirect `/rooms/ID` to `/rooms/ID/join` for non-owners
My URL to the room is `/rooms/0al-wxo-qji-dih` but is `/rooms/0al-wxo-qji-dih/join` for guests. I could imagine that many owners will just copy their URL - which will show an error for any other user/guest. This would mimic the behavior of V2 and might therefore even be necessary to guarantee compatibility.

### LINKED ISSUE #4201: V3: Show what the "Copy" button would copy
V2 showed what you'd copy with the "Copy" button.
image

In V3 there is only a "Copy" button.
image

I suggest to rename it to "Copy link" or (better IMHO) to show the link again.

### PR DESCRIPTION
**Pull request by farhatahmad**
## Description

- Redirect /rooms/ID to /rooms/ID/join for non room owners (closes #4200)
- Increase duration of toasts (closes #4199)
- Fixed broken presentation delete modal button (closes #4195)
- Reword copy button (closes #4201)
- Added missing toast strings (closes #4186)
- Added new languages

### CODE REVIEW

File: app/javascript/components/rooms/room/Room.jsx [hunk 1]
```diff
@@ -21,8 +23,14 @@ function copyInvite() {
...
   const { t } = useTranslation();
   const { friendlyId } = useParams();
-  const { data: room } = useRoom(friendlyId);
+  const { isError, data: room, error } = useRoom(friendlyId);
   const startMeeting = useStartMeeting(friendlyId);
+  const location = useLocation();
+
+  // Custom logic to redirect from Rooms page to join page if this isnt the users room and they're not allowed to view it
+  if (isError && error.response.status === 403) {
+    return <Navigate to={`${location.pathname}/join`} />;
```
Comment (farhatahmad) [inline app/javascript/components/rooms/room/Room.jsx:32]: Anyone know a better way to join paths properly in js?

File: app/javascript/App.jsx [hunk 2]
```diff
@@ -36,6 +36,9 @@ export default function App() {
       </Container>
       <Toaster
         position="bottom-right"
+        toastOptions={{
+          duration: 3000,
```
Comment (farhatahmad) [inline app/javascript/App.jsx:40]: Increased by 1 sec

File: app/javascript/hooks/queries/rooms/useRoom.jsx [hunk 3]
```diff
@@ -5,5 +5,8 @@ export default function useRoom(friendlyId) {
   return useQuery(
     ['getRoom', { friendlyId }],
     () => axios.get(`/rooms/${friendlyId}.json`).then((resp) => resp.data.data),
+    {
+      retry: false,
+    },
```
[the remark this answers is not in the document]
Comment (scouillard) [reply app/javascript/hooks/queries/rooms/useRoom.jsx:10]: Not sure of the impact of having 0 retries, I'm down to try it.
Isn't the redirecting somewhat of an edge case tho?
Can we get this to 1 retry - have some grace without impacting the UX much?

**Review by scouillard — DISMISSED** [review]

File: app/javascript/hooks/queries/rooms/useRoom.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (farhatahmad) [reply app/javascript/hooks/queries/rooms/useRoom.jsx:10]: Yeah let me try that

[the remark this answers is not in the document]
Comment (farhatahmad) [reply app/javascript/hooks/queries/rooms/useRoom.jsx]: I've set it to 1 - I think the underlying problem is that there's no placeholders for room name and last session. Once that's done, the UI will look better

**Review by scouillard — APPROVED** [review]

**Pull request merged**